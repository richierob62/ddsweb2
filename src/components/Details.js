import React from 'react';

const Details = () => {
    return (
        <div className="Details">
            <h1>This is the Details</h1>
        </div>
    );
}

export default Details;