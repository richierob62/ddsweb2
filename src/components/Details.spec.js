import React from 'react';
import Details from './Details';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';

it('renders without crashing', () => {
  const el = shallow(<Details />);
  expect(shallowToJson(el)).toMatchSnapshot();
});
