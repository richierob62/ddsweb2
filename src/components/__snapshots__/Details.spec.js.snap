exports[`test renders without crashing 1`] = `
<div
  className="Details">
  <h1>
    This is the Details
  </h1>
</div>
`;
